components/server/ee/src/prebuilds/github-app.ts

```diff
@@ -349,24 +352,39 @@ export class GithubApp {
         return this.config.hostUrl.with({ pathname: '/button/open-in-gitpod.svg' }).toString();
     }
 
-    protected async findProjectOwner(cloneURL: string): Promise<{user: User, project?: Project} | undefined> {
-        // Project mode
-        //
-        const project = await this.projectDB.findProjectByCloneUrl(cloneURL);
-        if (project) {
-            const owner = !!project.userId
-                ? { userId: project.userId }
-                : (await this.teamDB.findMembersByTeam(project.teamId || '')).filter(m => m.role === "owner")[0];
-            if (owner) {
-                const user = await this.userDB.findUserById(owner.userId);
-                if (user) {
-                    return { user, project}
-                }
+    /**
+     * Finds the relevant user account to create a prebuild with.
```

Comment:
This function does not properly handle projects which belong to a user account (not under a team).
In that case the function returns no users, which results in no prebuild being run.

Reply:
Thanks for the thorough testing! Fixed and commented.